ci/libnss_whatami.c

```diff
@@ -20,11 +20,12 @@
 #include <nss.h>
 #include <pwd.h>
 #include <string.h>
+#include <stdlib.h>
 
 enum nss_status
 _nss_whatami_getpwnam_r(const char *name, struct passwd *result, char *buffer, size_t buflen, int *errnop)
 {
-	if (strcmp(name, "whatami") == 0) {
+	if (strcmp(name, "whatami") == 0 || strncmp(name, "am_i_", 5) == 0) {
```

Comment:
since we're accepting more names, should we set `result->pw_name = name`? it probably doesn't matter for this test but maybe we won't regret it later.